lib/android.js

```diff
@@ -1057,7 +1057,7 @@ export function getArtClassSpec (vm) {
       const methodsArray = readArtArrray(object, offsetMethods, mInfo.artArrayLengthSize);
       const methodsArraySize = methodsArray.length;
       for (let offset = offsetMethods; offset !== MAX_OFFSET; offset += 4) {
-        if (object.add(offset).readU16() == methodsArraySize) {
+        if (object.add(offset).readU16() === methodsArraySize) {
```

Comment:
Please test this. During my testing I had === too but the equality check failed for some reason. It only worked with ==.